app-support/wifi/wifi.c

```diff
@@ -7,7 +7,7 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(golioth_wifi, LOG_LEVEL_DBG);
 
-#include <net/golioth/wifi.h>
+#include <app-support/wifi.h>
```

Comment:
So far we use `_` instead of `-` in directory names, I would prefer to keep that consistent and rename `app-support` to `app_support`.